README.md

```diff
@@ -42,3 +43,22 @@ iv. Run `node app.js`
 
 v. Begin developing ✌️
 
+## File structure
+```
+├── ...
+├── config.json
+│   ├── ℹ️ This houses configuration for beamng.cluster itself.
+│   ├──    This is edited and read by it.
+│   └──    You should not update this manually, unless you need to.
```

Comment:
from `│   └──    You should not update this manually, unless you need to.`

to `│   └──    ⚠️ You should not update this manually, unless you need to.`